Detections/SigninLogs/AuthenticationAttemptfromNewCountry.yaml

```diff
@@ -22,16 +22,32 @@ relevantTechniques:
 tags:
   - AADSecOpsGuide
 query: |
-  let known_locations =
-    union isfuzzy=True AADNonInteractiveUserSignInLogs, SigninLogs
-    | where TimeGenerated between(ago(14d)..ago(1d))
-    | where ResultType == 0
-    | summarize by Location;
-    union isfuzzy=True AADNonInteractiveUserSignInLogs, SigninLogs
-    | where TimeGenerated > ago(1d)
-    | where ResultType != 50126
-    | where Location !in (known_locations)
-    | extend LocationDetails_dynamic = column_ifexists("LocationDetails_dynamic", "")
+  // Combine two sign-in logs tables, AADNonInteractiveUserSignInLogs and SigninLogs
+  let CombinedSignInLogs = union isfuzzy=True AADNonInteractiveUserSignInLogs, SigninLogs;
+  // Fetch Azure IP address ranges data from a JSON file hosted on GitHub
+  let AzureRanges = externaldata(changeNumber: string, cloud: string, values: dynamic)
```

Comment:
@praveenthepro Please update the PR notes on why we require this improvement. The query concerns about a new location and not IPs. Thus, wanted to know you views on why this change is required.